src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -207,12 +216,17 @@ class ReplayWall implements Wall {
 }
 
 function createReactDevTools(
+  reactDevToolsInlineModule: any,
   annotations: Annotation[],
   currentPoint: ExecutionPoint,
   enablePicker: (opts: NodePickerOpts) => void,
   disablePicker: () => void,
   onShutdown: () => void
-) {
+): ReactDevToolsType {
+  const createBridge: typeof createBridgeType = reactDevToolsInlineModule.createBridge;
+  const createStore: typeof createStoreType = reactDevToolsInlineModule.createStore;
+  const initialize: typeof initializeType = reactDevToolsInlineModule.initialize;
```

Comment:
TypeScript seems weird to me. When I used `import type` I didn't have to also use `typeof` but it seems that makes it happy?

Reply:
Interestingly, even though you're using `import type`, `createBridgeType` etc. refer to the functions, not to their types. I'm surprised that Typescript doesn't complain about using `import type` for something that isn't a type but I guess it makes sense when you want to import a function's (or object's) type without creating a runtime dependency and the exporting module doesn't export the types separately.

Reply:
Interesting. Is there a better way to type these functions?

I guess I could just `declare createBridgeType` in the `.d.ts` module

Reply:
You could add `type createBridgeType = typeof createBridge;` to the `.d.ts` file.
But an even easier way would be to write `reactDevToolsInlineModule: typeof import("react-devtools-inline/frontend")` and then remove the explicit types from the functions that you extract from `reactDevToolsInlineModule`.

Reply:
Ahhhh, _this_ (`reactDevToolsInlineModule: typeof import("react-devtools-inline/frontend")`) is what I was looking for but didn't know how to do.